plugins/auth/index.js

```diff
@@ -95,12 +97,13 @@ const authPlugin = {
          * Generates a profile for storage in cookie and jwt
          * @method generateProfile
          * @param  {String}        username   Username of the person
+         * @param  {String}        scmUserId  User ID in the SCM
          * @param  {String}        scmContext Scm to which the person logged in belongs
          * @param  {Array}         scope      Scope for this profile (usually build or user)
          * @param  {Object}        metadata   Additonal information to tag along with the login
          * @return {Object}                   The profile to be stored in jwt and/or cookie
          */
-        server.expose('generateProfile', (username, scmContext, scope, metadata) => {
+        server.expose('generateProfile', (username, scmUserId, scmContext, scope, metadata) => {
```

Comment:
Would prefer something like below so order no longer matters (especially since you're inserting in the middle of the params)
```suggestion
        server.expose('generateProfile', ({ username, scmUserId, scmContext, scope, metadata }) => {
```

Reply:
https://github.com/screwdriver-cd/screwdriver/pull/2859